core/src/Microsoft.Identity.Core.csproj

```diff
@@ -9,21 +9,12 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.3;win81;MonoAndroid8.1;Xamarin.iOS10</TargetFrameworks>        
```

Comment:
This was used for msbuild /t:pack, but that route won't work, so I moved all this package metadata to a nuspec file

Reply:
Thanks for all the additional comments and explanation throughout...very helpful

Reply:
How does the compiler know that it needs to generate DLLs for core for all these platforms if the TargetFrameworks property does not appear?

Reply:
The <TargetFrameworks> was not removed, you can see it lower down. The GitHub diff view mistakenly thinks I deleted it and added it back lower down (now on line 12 in the Unified Diff view)